tools/Customization/DevHome.Customization/ViewModels/DevDriveInsightsViewModel.cs

```diff
@@ -14,12 +14,12 @@
 using DevHome.Customization.Models;
 using DevHome.Customization.ViewModels.DevDriveInsights;
 using DevHome.Customization.Views;
-using Microsoft.Internal.Windows.DevHome.Helpers;
+using Microsoft.UI.Dispatching;
 using Serilog;
 
 namespace DevHome.Customization.ViewModels;
 
-public partial class DevDriveInsightsViewModel : ObservableObject, IRecipient<DevDriveOptimizedMessage>, IRecipient<DevDriveTrustedMessage>
+public partial class DevDriveInsightsViewModel : ObservableObject, IRecipient<DevDriveOptimizedMessage>, IRecipient<DevDriveTrustedMessage>, IRecipient<DevDriveOptimizingMessage>

```

Comment:
I wonder if in the future we should start consolidating these messages by making them more generic. I'm just thinking about the case where we end up with like 5 more or something like that. If we consolidate them maybe we end up with one message class and then the receiving class can figure out the data based on some query method where we pass in an enum or something. Just something to think about is all.